README.rst

```diff
@@ -25,7 +25,7 @@ and `Lightning <https://www.pytorchlightning.ai>`_ for deep learning, and
 
 Quick links
 ===========
-* Install renate with `pip install renate` or look at `these instructions <https://renate.readthedocs.io/en/latest/getting_started/install.html>`_
+* Install renate with ``pip install renate`` or look at `these instructions <https://renate.readthedocs.io/en/latest/getting_started/install.html>`_
 * Examples for `local training <https://renate.readthedocs.io/en/latest/examples/train_mlp_locally.html>`_ and `training on Amazon SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_.
 * `Documentation <https://renate.readthedocs.io>`_
 * `Supported Algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_
```

Comment:
```suggestion
* `Installation <https://renate.readthedocs.io/en/latest/getting_started/install.html>`_
* Examples for `local training <https://renate.readthedocs.io/en/latest/examples/train_mlp_locally.html>`_ and `training on Amazon SageMaker <https://renate.readthedocs.io/en/latest/examples/train_classifier_sagemaker.html>`_.
* `Documentation <https://renate.readthedocs.io>`_
* `Supported Algorithms <https://renate.readthedocs.io/en/latest/getting_started/supported_algorithms.html>`_

Quick Installation
=============
You can install it via pip from PyPI.

.. code-block:: bash

    pip install renate
    
For other use cases, please check our `installation guide <https://renate.readthedocs.io/en/latest/getting_started/install.html>`_.
```

As I suggested to Giovanni, I think would be great to give this more visibility by giving it it's own section and codeblock that allows to copy paste by button.